src/category_theory/limits/shapes/strong_epi.lean

```diff
@@ -148,6 +174,25 @@ strong_epi_category.strong_epi_of_epi _
 lemma strong_mono_of_mono [strong_mono_category C] (f : P ⟶ Q) [mono f] : strong_mono f :=
 strong_mono_category.strong_mono_of_mono _
 
+lemma strong_epi_of_is_split_epi (f : P ⟶ Q) [is_split_epi f] : strong_epi f :=
```

Comment:
This should already follow from the implications split_epi -> regular_epi -> strong_epi shown in regular_mono.lean, right?

Reply:
Thanks for pointing this!